.github/workflows/deploy.yml

```diff
@@ -108,18 +108,38 @@ jobs:
           echo "application_endpoint=$APPLICATION_ENDPOINT" >> $GITHUB_OUTPUT
           echo "## Application Endpoint" >> $GITHUB_STEP_SUMMARY
           echo "<$APPLICATION_ENDPOINT>" >> $GITHUB_STEP_SUMMARY
-      - id: api_endpoint
+      - id: api_name
         run: |
-          API_ENDPOINT=$(./scripts/output.sh app-api ApiGatewayRestApiUrl $STAGE_PREFIX$branch_name)
-          echo "API_ENDPOINT=$API_ENDPOINT" >> $GITHUB_ENV    
-      - id: cognito_user_pool_id
+          API_NAME=$(./scripts/output.sh app-api ApiGatewayRestApiName $STAGE_PREFIX$branch_name)
+          echo "api_name=$API_NAME" >> $GITHUB_OUTPUT
+      - id: branch_name
+        run: |
+          echo "branch_name=$branch_name" >> $GITHUB_OUTPUT
+      - id: cognito_user_pool_client_id
+        run: |
+          COGNITO_USER_POOL_CLIENT_ID=$(./scripts/output.sh ui-auth UserPoolClientId $STAGE_PREFIX$branch_name)
+          echo "cognito_user_pool_client_id=$COGNITO_USER_POOL_CLIENT_ID" >> $GITHUB_OUTPUT
+      - id: cognito_user_pool_id_encrypted
         run: |
           COGNITO_USER_POOL_ID=$(./scripts/output.sh ui-auth UserPoolId $STAGE_PREFIX$branch_name)
-          echo "COGNITO_USER_POOL_ID=$COGNITO_USER_POOL_ID" >> $GITHUB_ENV      
+          COGNITO_USER_POOL_ID_ENCRYPTED=$(echo -n "$COGNITO_USER_POOL_ID" | openssl enc -e -aes-256-cbc -a -pbkdf2 -iter 10000 -k "${{ secrets.CODE_CLIMATE_ID }}")
```

Comment:
I have to encrypt this secret because GitHub automatically detects a value in it that can't be outputted. That was preventing being able to pass the secret to another job.